snykTask/src/__tests__/test-task-args.ts

```diff
@@ -103,6 +104,38 @@ describe('TaskArgs.setMonitorWhen', () => {
   });
 });
 
+describe('TaskArgs.severitySettings', () => {
```

Comment:
`TaskArgs.validate` naming would be consistent with other group of tests, where they're grouped by a function name (e.g. `TaskArgs.setMonitorWhen`).